test/resources/oauth_proxy_auth.go

```diff
@@ -0,0 +1,250 @@
+package resources
```

Comment:
@b1zzu I've put your code for oauth proxy authentication into this file, it should work with all product host URLs. I've added a step for permissions approval on first logins.